extractor/src/test/suite/scala/com/digitalasset/extractor/AuthSpec.scala

```diff
@@ -153,7 +153,7 @@ final class AuthSpec
       }
       .onComplete {
         case Success(tx) => val _ = expectedTxs += tx
-        case Failure(NonFatal(_)) => () // do nothing, the test will fail
+        case Failure(_) => () // do nothing, the test will fail
```

Comment:
futures only ever produce `NonFatal` afaik so this doesn’t change semantics and it shuts up the exhaustivenesschecker.